clang/test/InstallAPI/objcclasses.test

```diff
@@ -27,12 +27,40 @@ __attribute__((objc_exception))
 @interface Exception 
 @end
 
+
+//--- Foo.framework/PrivateHeaders/Foo_Private.h
+#import <Foo/Foo.h>
+
+@interface ClassWithIvars : Visible  {
```

Comment:
Please also add a test that has the reverse order of access control.

Reply:
Something like 

```
__attribute__((visibility("hidden")))
@interface Foo  {
@public 
int _ivar;
}
@
```
?